regolith/builders/coabuilder.py

```diff
@@ -65,14 +65,18 @@ def construct_global_ctx(self):
     def excel(self):
         rc = self.rc
         gtx = self.gtx
-        since_date = dt.date.today() - relativedelta(months=NUM_MONTHS)
+        if isinstance(self.rc.to_date, str):
+            since_date = dt.datetime.strptime(self.rc.to_date, '%Y-%m-%d').date() - relativedelta(months=NUM_MONTHS)
```

Comment:
This will need some documentation.  I will never remember whether to specify from date or to date!

Maybe catch either?  If people do ``to_date`` subtract NUM_MONTHS and if they do ``from_date`` then add it?  Not too sure.  I guess this is mostly for testing so maybe we just keep quiet about it and leave it as is

Reply:
The user can pass both `--to` and `--from` into the `build` right? I was thinking about dividing it into more cases, like if user passes in `--to` or `--from` or both options, but wasn't sure if I should do all of them. So for now I just left the `to_date` case in there. If you'd like I'll add all the other cases in.